### PR TITLE
All: headsign output 

### PR DESCRIPTION
**Pull request by pbougue**
Headsigns are printed in departures, arrivals, route_schedules, journeys (display_informations) and vehicle_journeys (stop_time)

No headsign on stop_schedules, as discussed with @stifoon 
JIRA: http://jira.canaltp.fr/browse/NAVP-235

### CODE REVIEW

**Comment by buildbotctp** [comment]
Can one of the admins verify this patch?

File: source/jormungandr/tests/ptref_tests.py [hunk 1]
```diff
@@ -495,3 +495,48 @@ def test_multiple_resource_uri_with_final_collection_uri(self):
...
+        assert 'error' not in response
+        journeys = get_not_null(response, 'journeys')
+        eq_(len(journeys), 1)
+        eq_(len(journeys[0]['sections']), 1)
+        eq_(journeys[0]['sections'][0]['display_informations']['headsign'], "A00")
+
+    def test_headsign_display_info_departures(self):
+        """test basic print of headsign in display informations for departures"""
+        response = self.query_region('stop_points/stop_point:stopB/departures?from_datetime=20120615T000000')
+        assert 'error' not in response
```
Comment (antoine-de) [inline source/jormungandr/tests/ptref_tests.py:521]: I don't think it's mandatory, should be checked by query_region

File: source/routing/tests/routing_api_test_data.h [hunk 2]
```diff
@@ -405,22 +405,18 @@ struct routing_api_data {
         b.sa("stopB", B.lon(), B.lat());
         if (activate_pt) {
             //we add a very fast bus (2 seconds) to be faster than walking and biking
-            b.vj("A")
-                ("stop_point:stopB", 8*3600 + 1*60, 8*3600 + 1 * 60)
-                ("stop_point:stopA", 8*3600 + 1 * 60 + 2, 8*3600 + 1*60 + 2)
+            b.vj("A")("stop_point:stopB", "08:01"_t)("stop_point:stopA", "08:01:02"_t)
```
Comment (antoine-de) [inline source/routing/tests/routing_api_test_data.h:408]: :+1:

File: source/time_tables/route_schedules.cpp [hunk 3]
```diff
@@ -288,7 +288,12 @@ route_schedule(const std::string& filter,
                     pbnavitia::Header* header = table->mutable_headers(j);
                     pbnavitia::PtDisplayInfo* vj_display_information = header->mutable_pt_display_informations();
                     auto vj = dt_stop_time.second->vehicle_journey;
-                    fill_pb_object(vj, d, vj_display_information, 0, now, action_period);
+                    fill_pb_object(vj, d, vj_display_information, dt_stop_time.second, nullptr,
+                                   0, now, action_period);
+                    vj_display_information->set_headsign(vj->name);
```
Comment (antoine-de) [inline source/time_tables/route_schedules.cpp:296]: I think  a comment might be nice here to explain why it is done only for the API

File: source/jormungandr/tests/ptref_tests.py (hunk 1 above)
Comment (pbougue) [inline source/jormungandr/tests/ptref_tests.py:521]: It's not checked, and not possible to add it, as sometimes we return a 200 with an error (no disruptions).

**Pull request merged**